package/base-files/files/sbin/opkg-keep

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+for file in /etc/sysupgrade.conf /usr/lib/opkg/status /etc/rc.local; do
+    cp $file $file.bak
+    echo $file.bak >> /etc/sysupgrade.conf
+done
+
+cat <<\EOF > /etc/rc.local
+#!/bin/sh
+opkg update && {
+    opkg install diffutils
```

Comment:
Thats less then ideal, you should be able to achieve the same effect by comparing two sorted lists. Also this lacks a check if diff is already present